Makefile

```diff
@@ -604,3 +604,49 @@ install-wwhrd:
 
 check-licenses: install-wwhrd
 	wwhrd check -f .wwhrd.yml
+
+tsbs: tsbs-build tsbs-generate-data tsbs-load-data tsbs-generate-queries tsbs-run-queries
+
+TSBS_SCALE := 4000
```

Comment:
@dmitryk-dk please add a comment that would roughly describe the amount of data generated by benchmark